docker/Dockerfile-run

```diff
@@ -1,6 +1,6 @@
 ####################################################################
 # base image contains all the dependencies we need to build the code
-FROM valhalla/valhalla:build-3.1.0 as builder
+FROM valhalla/valhalla:build-latest as builder
```

Comment:
this I don't understand.. how can we have fresh valhalla:run containers if this was always fixed to build-3.1.0 before?